flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/AbstractStreamOperator.java

```diff
@@ -227,11 +233,73 @@ public void setup(StreamTask<?, ?> containingTask, StreamConfig config, Output<S
 		}
 
 		this.runtimeContext = new StreamingRuntimeContext(this, environment, container.getAccumulatorMap());
+		this.mailboxExecutor = this.getContainingTask().getMailboxExecutor(config.getChainIndex());
 
 		stateKeySelector1 = config.getStatePartitioner(0, getUserCodeClassloader());
 		stateKeySelector2 = config.getStatePartitioner(1, getUserCodeClassloader());
 	}
 
+	/**
+	 * Allows downstream operators in the operator chain to process events.
+	 * <p>This method should be called whenever an operator would need to block the task thread to wait for output
```

Comment:
Suggestion: use new (empty) line before `<p>` and remove the closing `</p>`s. This "convention" is more common in the Flink codebase.

Reply:
Since we are slowly adopting newer Java versions, it's quite likely that we also switch to new javadoc with HTML5. I'd prefer to not revert back to HTML4 unless you think it is confusing to devs and users.

Reply:
I can't say much about newer Java version transition and if it would involve reformatting of all Java docs.
Afaik, HTML5 also allows non paired `<p>` tags. So the old "convention" could be still valid.

I'm not opinionated which style to choose, but I'm for consistence and more uniform style.

Reply:
Okay, I will remove it then. But FYI, HTML5 is as strict as XML, so you need to have a closing tag.